ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/representative-journey.ts

```diff
@@ -67,9 +67,8 @@ export class RepresentativeJourney extends JourneyBase {
       && model.clientAttendance !== undefined
       && model.hearingSuitability.answer !== undefined
       && model.room !== undefined
-      && model.computer !== undefined
-      && model.camera !== undefined;
```

Comment:
Why remove camera ?

Reply:
When answer for the computer in No then Camera will be Undefined